utils/test-utils/src/main/scala/pl/touk/nussknacker/test/WithConfig.scala

```diff
@@ -1,10 +1,24 @@
 package pl.touk.nussknacker.test
 
 import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.commons.io.IOUtils
+
+import java.nio.charset.StandardCharsets
+
+import scala.util.Try
 
 trait WithConfig {
 
-  protected lazy val config: Config = resolveConfig(ConfigFactory.load())
+  // Here is the place where we can change the entry config file
+  protected val configFilename: String = "application.conf"
+
+  protected lazy val config: Config = {
+    val config = Try(IOUtils.resourceToString(s"/$configFilename", StandardCharsets.UTF_8)).toOption
```

Comment:
Why this `Try().toOption`? It is always better to fail fast instead of postponing the problem to the place where you don't know what's going on

Reply:
Hmm, we can do that. But I saw that we have some tests without this file and exception was thrown in these tests..

Reply:
Hmm, I think that I'm getting your intention now. Before your change, was loaded default chain of configs, which essentially was an empty config for such a case. If we resign from usage of this chain and do everything by our own, we just check if file exists and if not, return empty config

Reply:
![image](https://github.com/user-attachments/assets/1541f353-7823-4260-907a-822615cd02ba)
Old version if applicaiton.conf doesn't exit in test resource creates empty / default config..

Reply:
My proposition:

```
  // Here is the place where we can change the entry config file
  protected val configFilename: Option[String] = None

  protected lazy val config: Config = {
    val config = configFilename
      .map(file => IOUtils.resourceToString(s"/$file", StandardCharsets.UTF_8))
      .map(ConfigFactory.parseString)
      .getOrElse(ConfigFactory.empty())

    resolveConfig(config.resolve())
  }
  ```
  WDYT?

Reply:
Looks good